CONTRIBUTING.md

```diff
@@ -224,9 +224,9 @@ Verify the screenshots and commit the changes.
 - `Shift + T` : Hide the toolbar
 
 ## Code Style
-Code style is ensured by [detekt](https://github.com/detekt/detekt). It runs automatically during the `check` phase but can also be executed by running `gradlew detekt`.
+Code style is ensured by [detekt](https://github.com/detekt/detekt). It runs automatically during the `check` phase but can also be executed by running `./gradlew detekt`.
 
-To auto fix problems run `gradlew detekt` again.
+To auto fix problems run `./gradlew detekt` again.
```

Comment:
Can you change this to `./gradlew detekt --auto-correct` and remove the again please?